slack_today_i_did/reports.py

```diff
@@ -110,3 +110,61 @@ def as_dict(self):
             'people_to_bother': self.people_to_bother,
             'time_run': (str(self.time_run) if self.time_run is not None else "")  # noqa: E501
         }
+
+
+class Sessions(object):
+    def __init__(self):
+        self.sessions = {}
+
+    def has_running_session(self, person: str) -> bool:
+        if person not in self.sessions:
+            return False
+
+        return self.sessions[person]['is_running']
+
+    def start_session(self, person: str, channel: str) -> None:
+        self.sessions[person] = {
+            'is_running': True,
+            'messages': [],
+            'channel': channel
+        }
+
+    def end_session(self, person: str) -> None:
+        if not self.has_running_session(person):
+            return
+
+        self.sessions[person]['is_running'] = False
+
+    def add_message(self, person: str, message: str) -> None:
+        if not self.has_running_session(person):
+            return
+
+        self.sessions[person]['messages'].append(message)
+
+    def get_entry(self, person: str) -> Dict[str, Any]:
+        if person not in self.sessions:
+            return {}
+
+        return self.sessions[person]
+
+    def retire_session(self, person: str, filename: str) -> None:
```

Comment:
Yep - I added it so that later on it would be possible to log a session to a file